implementations/rust/ockam/ockam_command/src/project/mod.rs

```diff
@@ -65,6 +77,8 @@ impl ProjectCommand {
             ProjectSubcommand::AddEnroller(scmd) => AddEnrollerCommand::run(opts, scmd),
             ProjectSubcommand::ListEnrollers(scmd) => ListEnrollersCommand::run(opts, scmd),
             ProjectSubcommand::DeleteEnroller(scmd) => DeleteEnrollerCommand::run(opts, scmd),
+            ProjectSubcommand::AddMember(scmd) => scmd.run(opts),
```

Comment:
Can this please look just like every other match statement for command?

Reply:
The conciseness of this style is nice but let's keep it consistent since we have a lot of new contributors exploring this part of the code.

@twittner let's file a `good first issue` to change to the more concise style across all commands.